presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -395,49 +460,17 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
         }
       }
       if (op.numSplits != 0) {
-        const auto statName = fmt::format(
-            "{}.{}.{}", op.operatorType, op.planNodeId, "numSplits");
+        const auto statName =
+            fmt::format("{}.{}.numSplits", op.operatorType, op.planNodeId);
         opOut.runtimeStats.emplace(
-            statName,
-            protocol::RuntimeMetric{
-                statName,
-                protocol::RuntimeUnit::NONE,
-                op.numSplits,
-                1,
-                op.numSplits,
-                op.numSplits});
+            statName, createProtocolRuntimeMetric(statName, op.numSplits));
       }
 
-      // If Velox's operator has spilling stats, then add them to the protocol
+      addRunningTimeOperatorMetrics(opOut, prestoTaskStats, op);
```

Comment:
Would it be better to replace `prestoTaskStats` with `taskRuntimeStats` here?

Reply:
Same comment as above. :)